integration_tests/configs/cronos-experimental-devnet.jsonnet

```diff
@@ -0,0 +1,33 @@
+local config = import 'default.jsonnet';
+
+config {
+  'cronos_777-1'+: {
+    'cmd-flags': '--unsafe-experimental',
+    'start-flags': '--trace',
+    config:: super['config'],
```

Comment:
is this line redundant?

Reply:
just keep same [experimental](https://github.com/crypto-org-chain/cronos/blob/main/scripts/cronos-experimental-devnet.yaml) config, since we don't have [mempool](https://github.com/crypto-org-chain/cronos/blob/main/integration_tests/configs/default.jsonnet#L8) version specified before

Reply:
what's the difference of `config:: super['config'],` with omitting this line?

Reply:
yes, to remove config from default

Reply:
I see, hidden field 👍